Source/WebCore/Scripts/generate-process-sync-data.py

```diff
@@ -513,14 +513,14 @@ def generate_process_sync_data_serialiation_in(synched_datas, document_synched_d
 
 
 def sort_data_lists(synched_datas):
-    type_set = set()
-    conditional_type_set = set()
+    type_set = []
+    conditional_type_set = []
 
     for data in synched_datas:
         if data.conditional is None:
-            type_set.add(data)
+            type_set.append(data)
         else:
-            conditional_type_set.add(data)
+            conditional_type_set.append(data)
 
     type_list = sorted(list(type_set), key=lambda data: data.fully_qualified_type)
     conditional_type_list = sorted(list(conditional_type_set), key=lambda data: data.fully_qualified_type)
```

Comment:
You probably don't want `sorted(list(` here any more.

Reply:
May as well.